### PR TITLE
Update v65.4.0

### PR DESCRIPTION
**Pull request by pseudoyim**

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-{% set version = "63.4.1" %}
+{% set version = "65.4.0" %}
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:1]: Can we build the latest v65.5.0 https://github.com/pypa/setuptools/blob/v65.5.0/CHANGES.rst#v6550

**Review by skupr-anaconda — APPROVED** [review]

**Review by varlackc — APPROVED** [review]
LGTM

File: recipe/meta.yaml (hunk 1 above)
Comment (Arishamays1) [reply recipe/meta.yaml:1]: Got it, will do after merging this one!

**Review by Arishamays1 — COMMENTED** [review]
LGTM

**Pull request merged**